go.mod

```diff
@@ -2,13 +2,15 @@ module github.com/giantswarm/cluster-test-suites
 
 go 1.21
 
+replace github.com/giantswarm/cluster-standup-teardown => github.com/giantswarm/cluster-standup-teardown v0.0.0-20240426074914-3870735834b4
```

Comment:
Currently required while testing this PR out. Once confirmed that this change works I can merge and release the initial version of `cluster-standup-teardown` and remove this replace directive.

Reply:
Done.